### PR TITLE
[NETPROD-3582] Added name query param in list certificates

### PR DESCRIPTION
**Pull request by guptado**
[NETPROD-3582](https://do-internal.atlassian.net/browse/NETPROD-3582)

[NETPROD-3582]: https://do-internal.atlassian.net/browse/NETPROD-3582?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: specification/resources/certificates/parameters.yml [hunk 1]
```diff
@@ -7,4 +7,14 @@ certificate_id:
     type: string
     format: uuid
     minimum: 1
-  example: 4de7ac8b-495b-4884-9a69-1050c6793cd6
\ No newline at end of file
+  example: 4de7ac8b-495b-4884-9a69-1050c6793cd6
+
+name:
```
Comment (andrewsomething) [inline specification/resources/certificates/parameters.yml:12]:
```suggestion
certificate_name:
```

File: specification/resources/certificates/certificates_list.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ description: >-
 parameters:
   - $ref: '../../shared/parameters.yml#/per_page'
   - $ref: '../../shared/parameters.yml#/page'
+  - $ref: "parameters.yml#/name"
```
Comment (andrewsomething) [inline specification/resources/certificates/certificates_list.yml:12]:
```suggestion
  - $ref: "parameters.yml#/certificate_name"
```

**Review by andrewsomething — DISMISSED** [review]
Small suggestion to use a more unique name for the parameter to prevent any future conflicts.

**Comment by guptado** [comment]
> Small suggestion to use a more unique name for the parameter to prevent any future conflicts.

Included the changes

**Review by andrewsomething — APPROVED** [review]
:+1:  LGTM!

**Pull request merged**